wdl/methods/phasing/PhysicalAndStatisticalPhasing.wdl

```diff
@@ -225,3 +232,39 @@ task FilterAndConcatVcfs {
         docker:"us.gcr.io/broad-dsp-lrma/lr-gcloud-samtools:0.1.2"
     }
 }
+
+
+# filter out singletons (i.e., keep MAC >= 2) and concatenate with deduplication
+task FilterPhasingInfo {
+
+    input {
+        File sv_vcf            # biallelic
+        File sv_vcf_tbi
+        String prefix
+    }
+
+    command <<<
+        set -euxo pipefail
+
+        # filter SV singletons
+        bcftools +setGT ~{sv_vcf} -- -t a -n u | bgzip > ~{prefix}.vcf.gz
```

Comment:
A comment would be useful to document the meaning of `-t a -n u`. Also `-Oz -o` is used elsewhere, rather than piping to `bgzip`. Please also format the command line and any line breaks as elsewhere.